src/Akka.Persistence.Azure/TableEntities/AllPersistenceIdsEntry.cs

```diff
@@ -1,61 +1,58 @@
 ﻿using System;
-using System.Collections.Generic;
-using Microsoft.Azure.Cosmos.Table;
+using Azure;
+using Azure.Data.Tables;
 
 namespace Akka.Persistence.Azure.TableEntities
 {
-    internal sealed class AllPersistenceIdsEntry
-        : ITableEntity
+    internal sealed class AllPersistenceIdsEntry 
     {
         private const string ManifestKeyName = "manifest";
         public const string PartitionKeyValue = "allPersistenceIdsIdx";
 
         // In order to use this in a TableQuery a parameterless constructor is required
-        public AllPersistenceIdsEntry()
+        public AllPersistenceIdsEntry(TableEntity entity)
```

Comment:
Instead of relying on the SDK deserialization code, we will use the EventHubs generic `TableEntity` serialization and populate our own data from it.